docs/concepts/workloads/vm.md

```diff
@@ -566,18 +566,24 @@ The following fields summarize a `VirtualMachine` resource's overall storage usa
 
 | Name | Description |
 |------|-------------|
-| `status.storage.committed` | The total storage space committed to this `VirtualMachine`. |
-| `status.storage.uncommitted` | The total storage space potentially used by this `VirtualMachine` |
-| `status.storage.unshared` | The total storage space occupied by this VirtualMachine that is not shared with any other `VirtualMachine`. |
+| `status.storage.other.used` | The total storage space used by all of the VM's non-disk files, ex. snapshots, logs, etc. |
```

Comment:
Are the "snapshots" here refer to VM snapshots or something else?

Reply:
Yes. Obviously this would be updated once we introduce VM snapshots as first-class objects.